websites/G/Glimesh/presence.ts

```diff
@@ -166,6 +142,26 @@ presence.on("UpdateData", async () => {
     presenceData.details = "Viewing Merch Store";
   } else if (locationHost == "support.glimesh.tv") {
     presenceData.details = "Viewing Support";
+  } else if (locationHost == "blog.glimesh.tv") {
+    presenceData.details = "Viewing Blogs";
+
+    if (locationPath.match("/posts")) {
+      const blogPost = document.querySelector("body > div > div > div > h1 > a").textContent;
+      
+      presenceData.details = "Reading" + (!await presence.getSetting("show_details") ? " a " : " ") + "Blog";
+      if (await presence.getSetting("show_details")) {
+        presenceData.state = blogPost;
+      }
```

Comment:
```suggestion
      if (await presence.getSetting("show_details")) presenceData.state = blogPost;
```

Reply:
Just those lines or do you want me to replicate that through out the entire script?

Reply:
Yes.